proto/enums/enums.proto

```diff
@@ -554,6 +577,10 @@ enum SuggestionCode {
     CONFIG_CHECK_PROFILE_SELECTION = 706;
     // Fix config API version or upgrade the skaffold binary
     CONFIG_FIX_API_VERSION = 707;
+    // Only the whitelisted validators are acceptable in skaffold-managed mode.
+    CONFIG_WHITELISTED_VALIDATORS = 708;
```

Comment:
Since this code is related to this [part](https://github.com/GoogleContainerTools/skaffold/pull/5953/files#diff-3fad7c6ab5eda79a7c6a50eaa8510ce14287bc2dddb03f3b02aa1c20166b7a96R42) do we plan to explain on what's actually will be supported on kpt and skaffold managed mode?

Reply:
yeah，I've added some todo comments in the code to mark what should be explained about kpt-managed and skaffold-managed mode.